cmd/gateway/s3x/gateway-s3x-utils.go

```diff
@@ -26,3 +29,24 @@ func getMinioObjectInfo(o *ObjectInfo) minio.ObjectInfo {
 		UserDefined: o.UserDefined,
 	}
 }
+
+// ******  FLEEK UTILS *************
+
+func pingHash(hash string) {
+	// PING hashes to Temporal and FLEEK gateway
+	go func () {
+		_, err := http.Get("https://temporal.gateway.cloud/ipfs/" + hash)
```

Comment:
Maybe this is okay for now, but from my experience, `http.Get()` by default doesn't have a timeout right? Maybe we want to have a timeout set, incase one of the gateways is malfunctioning and is taking forever to get the new hash, we could end up have a backlog of go routines running.

Other than that LGTM

Reply:
yeah, Janison says timeout is not relevant, as long as it hits the GW
i guess we can add retries later for the ping to be more robust